site/en/docs/privacy-sandbox/permissions-policy/index.md

```diff
@@ -95,7 +95,7 @@ In this example, the header origin list lets only your site (`self`) and `https:
 Permissions-Policy: &lt;feature&gt;=(&lt;token&gt;|&lt;origin(s)&gt;)
 ```
 
-A `Permissions-Policy` header in the response from the server is used to set the allowed origins for the feature. The header value can take a combination of tokens and strings of origins. The [available tokens](https://w3c.github.io/webappsec-permissions-policy/#structured-header-serialization) are `*` for all origins and `self` for same-origin. Separate multiple features in the header with a comma. Separate multiple origins in the origin list with a space between. 
+A `Permissions-Policy` header in the response from the server is used to set the allowed origins for the feature. The header value can take a combination of tokens and strings of origins. The [available tokens](https://w3c.github.io/webappsec-permissions-policy/#structured-header-serialization) are `*` for all origins and `self` for same-origin. Separate multiple features in the header with a comma. Separate multiple origins in the origin list with a space between. If the origin is a cross-origin, the iframe tag must include the `allow` attribute along with being added to the origin list. 
```

Comment:
I'm a bit confused reading this addition -- I'm not sure "an origin" can be "a cross-origin." Is there another way to frame this update?

Reply:
I tried to parse this out a bit for my clarity -- would "request" work as the noun associated with "cross-origin"?